src/App.js

```diff
@@ -29,30 +29,41 @@ class App extends Component {
       this.getActivePlayers(allPlayers)
     }
   }
-  
+
+  showFrontOrBack = (sideToShow) => {
+      this.setState({ showCardSide: sideToShow})
+  }
+
+  setCurrentPlayer = (chosenPlayer) => {
+    this.setState({ chosenPlayer: chosenPlayer })
+  }
+
   getActivePlayers(players) {
     const activePlayers = players.filter(player => player.Status === 'Active')
     this.setState({ activePlayers: activePlayers })
   }
 
-  setPlayersName() {
-    this.state.activePlayers.map(player => {
-      const fullName = `${player.LastName}, ${player.FirstName}`
-      player.name = fullName
-      return fullName
-    })
-  }
-
   render() {
     return (
       <div className="app">
         <header className="header">WHO'S THAT CUB?</header>
-        <Find players={this.state.activePlayers} setPlayersName={this.setPlayersName}/>
+          <Find 
+            players={this.state.activePlayers} 
+            setCurrentPlayer={this.setCurrentPlayer} 
+            showFrontOrBack={this.showFrontOrBack}
+            showCardSide={this.state.showCardSide}
+            chosenPlayer={this.state.chosenPlayer}
+          />
```

Comment:
These are not all used (or passed through) in <Find> - remove `showCardSide` and `chosenPlayer`.